docs/docs/user_guide/sidebar/common/workspaces/accessing_a_workspace.md

```diff
@@ -80,3 +80,14 @@ Connection to workspace is restricted to specific CIDR block.
 - Check if your public IP is covered by the restricted CIDR block of the workspace.
 - Check if workspace type configuration has hard-coded value in field 'AccessFromCIDRBlock'. (Admin only)
 - If you're using VPN, your public IP address might change. Try disconnect VPN, and then connect to workspace.
+
+### Example Repositories
+All workspaces come with some examples to get you started with accessing data using the 
+Service Workbench and using the PIC-SURE API. You can find these repositories in the following
+locations, depending on instance type:
+
+- **EC2 Linux**: `/home/ec2-user/repos/`
```

Comment:
Let's check with out BSAs if the folder name `repos` conveys enough to the end user.

Reply:
Simran: `example_workflows`